.github/workflows/turbo-rails.yml

```diff
@@ -3,7 +3,7 @@ name: turbo-rails
 # Note: turbo-rails often returns an ActionDispatch::Response::RackBody for the
 # body. Also, Rack::BodyProxy or Sprockets::Asset.
 
-on: [push, pull_request, workflow_dispatch]
+on: [push, review_requested, ready_for_review, workflow_dispatch]
```

Comment:
personally I think this is better in general for draft PR workflows but can remove after it's passing tests in local